### PR TITLE
make sure we are removing files from the correct cache directory

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

**Comment by frett** [comment]
sure, travis is passing, guessing it's something with your local environment. maybe some of the recent changes to the gto support library.

**Comment by gyasistory** [comment]
Let me try again

**Comment by gyasistory** [comment]
Sorry I meant to say it is crashing and throwing the following error.

`E/AndroidRuntime: FATAL EXCEPTION: AEMDownloadManger #1
    Process: org.keynote.godtools.android.debug, PID: 23432
    java.lang.NullPointerException
        at com.annimon.stream.Objects.requireNonNull(Objects.java:127)
        at com.annimon.stream.Stream.of(Stream.java:85)
        at org.cru.godtools.articles.aem.service.AEMDownloadManger.cleanOrphanedFiles(AEMDownloadManger.java:452)
        at org.cru.godtools.articles.aem.service.AEMDownloadManger$CleanOrphanedFilesTask.runTask(AEMDownloadManger.java:697)
        at org.cru.godtools.articles.aem.service.AEMDownloadManger$UniqueTask.run(AEMDownloadManger.java:595)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
        at java.lang.Thread.run(Thread.java:764)`

**Comment by frett** [comment]
k, I'll take a look

**Comment by gyasistory** [comment]
I am running on Kotlin 1.2.71 are you on the same.

**Comment by frett** [comment]
yeah, I'll dig in shortly, finishing up the AndroidX upgrade currently

**Comment by frett** [comment]
I see the issue, `listFiles()` can return null if the directory doesn't exist. I'll fix it

**Comment by frett** [comment]
@gyasistory good catch
